Sources/xcproj/PBXGroup.swift

```diff
@@ -25,19 +16,26 @@ final public class PBXGroup: PBXFileElement {
     ///   - sourceTree: group source tree.
     ///   - name: group name.
     ///   - path: group path.
+    ///   - wrapsLines: should the IDE wrap lines for files in the group?
     ///   - usesTabs: group uses tabs.
     public init(children: [String],
```

Comment:
Same here. Would you mind adding them? 😬 
I'll go through all the files on the project and make sure they are properly documented.